css/style.css

```diff
@@ -1,7 +1,309 @@
+html {
+	box-sizing: border-box; /*padding и border включаются в общую ширину и высоту элемента*/
+ }
+
+*, *::before, *::after {
+	box-sizing: inherit;
+}
+
+html. body {
+	height: 100%;
+}
+
+body {
+	margin: 0 auto; 
+	color: #E8EEE4; 
+	font: normal 22px/26px MuseoSansCyrl;
+	overflow: scroll;
+}
+
+.wrap {
+	height: 150vh;
+	background: url(../img/iswanto-arif-448003-unsplash.png) no-repeat center; /*фоновая картинка*/
+	background-size: cover; /*картинка полностью покрывает блок при любых размерах*/
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Project-Bali/pullRequest?prid=5455735)